[ML] Fix date marker formatting in revert model snapshot flyout

darnautov opened the pull request:
## Summary

Fixes https://github.com/elastic/kibana/issues/98649

![Jul-06-2021 11-38-27](https://user-images.githubusercontent.com/5236598/124578714-bf1b8800-de4e-11eb-947c-668e96f3af56.gif)

Note: The marker body gets aligned correctly only after resizing the page, cc @markov00  

### Checklist

- [ ] This renders correctly on smaller devices using a responsive layout. (You can test this [in your browser](https://www.browserstack.com/guide/responsive-testing-on-local-server))
- [ ] This was checked for [cross-browser compatibility](https://www.elastic.co/support/matrix#matrix_browsers)

darnautov committed:
[ML] add markerBody

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/136486/)
* Commit: 516cde6a5c3e4b41ea3b90b0b588d741564339cd
* Storybooks not built yet
* This comment will update when the build is complete

### Failed CI Steps
* [Setup Build Environment and Dependencies](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/136486/execution/node/661/log)
* [Setup Build Environment and Dependencies](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/136486/execution/node/666/log)

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

cc @darnautov

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/136486/","number":"136486","commit":"516cde6a5c3e4b41ea3b90b0b588d741564339cd"}]}
PIPELINE-->

jgowdyelastic reviewed (approved):
LGTM

darnautov commented:
@elasticmachine merge upstream

kibanamachine committed:
Merge branch 'master' into ml-98649-fix-date-marker

Commit:
[Canvas] Move away from lib/workpad_service (#104183)

* Move away from lib/workpad_service

* Adds stubs

* Fix types. Swap fetching zip to workpad service

* Fix types

Co-authored-by: Kibana Machine <42973632+kibanamachine@users.noreply.github.com>

lcawl committed:
[DOCS] Fixes links to anomaly detection overview (#104350)

dasansol92 committed:
[Security Solution][Endpoint] Event filters text adjustments to be consistent with trusted apps (#104438)

* Text adjustments to be consistent with trusted apps

* Changes flyout submit button text

darnautov committed:
[Transform] Fix aggregation name override for the `top_metrics` aggs (#104446)

* [Transform] Fix aggName for the top_metrics agg

* [Transform] update comment

cavokz committed:
CCS Cypress integration (#103941)

* Add CCS Cypress test runner

* Split flow for CCS Cypress tests

* Make esArchiver load data onto the remote cluster

* Add CCS specific rules with customizable remote name

* Allow overriding @kbn/dev-utils's CA_CERT_PATH

* Add CCS related docs

Co-authored-by: Gloria Hornero <snootchie.boochies@gmail.com>

XavierM committed:
[RAC] ALerts table in observability (#103270)

Closes #98611

## Summary

Add alerts table in Observability => 

![image](https://user-images.githubusercontent.com/189600/123854490-c68ddf00-d8ec-11eb-897e-2217249d5fba.png)


### Risk Matrix

Delete this section if it is not applicable to this PR.

Before closing this PR, invite QA, stakeholders, and other developers to identify risks that should be tested prior to the change/feature release.

When forming the risk matrix, consider some of the following examples and how they may potentially impact the change:

| Risk                      | Probability | Severity | Mitigation/Notes        |
|---------------------------|-------------|----------|-------------------------|
| Multiple Spaces&mdash;unexpected behavior in non-default Kibana Space. | Low | High | Integration tests will verify that all features are still supported in non-default Kibana Space and when user switches between spaces. |
| Multiple nodes&mdash;Elasticsearch polling might have race conditions when multiple Kibana nodes are polling for the same tasks. | High | Low | Tasks are idempotent, so executing them multiple times will not result in logical error, but will degrade performance. To test for this case we add plenty of unit tests around this logic and document manual testing procedure. |
| Code should gracefully handle cases when feature X or plugin Y are disabled. | Medium | High | Unit tests will verify that any feature flag or plugin combination still results in our service operational. |
| [See more potential risk examples](https://github.com/elastic/kibana/blob/master/RISK_MATRIX.mdx) |


### For maintainers

- [ ] This was checked for breaking API changes and was [labeled appropriately](https://www.elastic.co/guide/en/kibana/master/contributing.html#kibana-release-notes-process)

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/136602/)
* Commit: 1441b94adae980f9fa1d88cd3be1b6a31d65fff2
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-104394/1441b94adae980f9fa1d88cd3be1b6a31d65fff2)

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/eb57dd4a7e35ecd3db49b0768adf3a890f33e41b) | [after](https://github.com/elastic/kibana/commit/1441b94adae980f9fa1d88cd3be1b6a31d65fff2) | diff |
| --- | --- | --- | --- |
| `ml` | 5.9MB | 5.9MB | -4.0B |

### History
* :broken_heart: [Build #136486](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/136486/) failed 516cde6a5c3e4b41ea3b90b0b588d741564339cd

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

cc @darnautov

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/136486/","number":"136486","commit":"516cde6a5c3e4b41ea3b90b0b588d741564339cd"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/136602/","number":"136602","commit":"1441b94adae980f9fa1d88cd3be1b6a31d65fff2"}]}
PIPELINE-->

michaelolo24 committed:
[Security Solution][Analyze Event] Fix resolver range slider (#104475)

angorayc committed:
[Security Solution] Remove total count for events table on hosts and details page (#104114)

* remove total count for events table on hosts and details page

* hide events count when analyzing an event

* fix unit tests

Co-authored-by: Kibana Machine <42973632+kibanamachine@users.noreply.github.com>

angorayc committed:
fix links for rule monitoring table (#104473)

alvarezmelissa87 committed:
[ML] Data Frame Analytics creation: ensure included fields selection table is shown when no docs contain all mapped fields (#104191)

* use indexPatternFields if explain error due to missing values in docs

* fix types

brianseeders committed:
[Docs] Add a tutorial for how to open a pull request (#103697)

FrankHassanabad committed:
Fixes indeterminism with the cy.intercept by moving it upwards above the clicks (#104501)

## Summary

Fixes indeterminism with cypress by moving the `cy.intercept('PATCH', '/api/timeline').as('updateTimeline');` above where the click is happening.

Backport to 7.14.0 to hopefully stabilize it better as well.

You can make the indeterminism deterministic by adding (for testing only), a `cy.wait(5000)` like so:

```ts
// Keep clicking on the disable all button until the first element of all the elements are no longer checked.
// In cases where the click handler is not present on the page just yet, this will cause the button to be clicked
// multiple times until it sees that the click took effect. You could go through the whole list but I just check
// for the first to be unchecked and then assume the click was successful
cy.root()
  .pipe(($el) => {
    $el.find(TIMELINE_ROW_RENDERERS_DISABLE_ALL_BTN).trigger('click');
    return $el.find(TIMELINE_ROW_RENDERERS_MODAL_ITEMS_CHECKBOX).first();
  })
  .should('not.be.checked');
 cy.wait(5000); // <--- Temp addition for testing to ensure our intercept works above this. If the intercept is below this then we see the same indetermism behavior but deterministically.
``` 

### Checklist

- [x] [Unit or functional tests](https://www.elastic.co/guide/en/kibana/master/development-tests.html) were updated or added to match the most common scenarios

tsullivan committed:
[Reporting] Add warning logs about CSV export type being deprecated (#104025)

* add deprecation logging to csv export type

* fix payload.isDeprecated

Co-authored-by: Kibana Machine <42973632+kibanamachine@users.noreply.github.com>

neptunian committed:
[Monitoring] update SM rule template variables (#104176)

* continue to support shardIndices template variable

* add unit test for large shard size alert

* change template variables in CCR rule

* add CCR rule unit test

* fix internationalization

Co-authored-by: Kibana Machine <42973632+kibanamachine@users.noreply.github.com>

jbudz committed:
[target/windows] Fix KBN_PATH_CONF expansion in bin/*.bat (#104517)

When setting the environment variable KBN_PATH_CONF in Windows, paths
containing special characters will be expanded and interpreted, instead
of being treated as a string.  This wraps `%KBN_PATH_CONF%` in quotes to
prevent this.

Testing:
1) set "KBN_PATH_CONF=C:\With space\special()%$characters\kibana\config"
2) bin\kibana*.bat should run

Commit:
[kbn/optimizer] maintain bundle ref support when using --filter (#104512)

Co-authored-by: spalger <spalger@users.noreply.github.com>

kpollich committed:
[Integrations UI] Fix missing agent flyout on package policy page (#104493)

* Fix missing agent flyout on package policy page

Handle query string changes if user clicks "add agent" link in toast
content while still on the package policy listing page.

Closes #103266

* Unbind history listener in useEffect

smith committed:
Remove schema heading from schema settings (#104520)

Also add a story.

Fixes #103753.

machadoum committed:
Fix long data for investigation not truncate on the rule details page (#104428)

Author: Pablo Machado <machadoum@gmail.com>
Co-authored-by: Pablo Machado <machadoum@gmail.com>

michaelolo24 committed:
[Security Solution] - Fix flyout pane z index (#104465)

patrykkopycinski committed:
[Osquery] Fix issue with plugin initialization (#100208)

Commit:
[apm-config-loader] refactor config loading (#104197)

Co-authored-by: spalger <spalger@users.noreply.github.com>

FrankHassanabad committed:
Removes older configuration that is no longer in use, removes un-used configuration keys (#104540)

## Summary

* Removes `kibana.yml` (configuration) keys that are not in use 
* Removes an older configuration schema file from before the merge of endpoint with security solutions

Unused schema file was:
```
server/endpoint/config.ts
```

Unused keys removed:

```
xpack.securitySolution.alertResultListDefaultDateRange
xpack.securitySolution.validateArtifactDownloads
```

### Checklist

- [x] [Unit or functional tests](https://www.elastic.co/guide/en/kibana/master/development-tests.html) were updated or added to match the most common scenarios

mistic committed:
chore(NA): remove legacy dev dep on @kbn/test from xpack package.json (#104476)

jgowdyelastic committed:
[ML] Removing broken data recognizer test (#104387)

jgowdyelastic committed:
[ML] Adding tests for ML's field caps endpoint (#104315)

* [ML] Adding tests for ML's field caps endpoint

* fixing typos

thomheymann committed:
Upgrade nodemailer (#104122)

Co-authored-by: Kibana Machine <42973632+kibanamachine@users.noreply.github.com>

thomheymann committed:
Upgrade prismjs (#104131)

Co-authored-by: Kibana Machine <42973632+kibanamachine@users.noreply.github.com>

Commit:
[Fleet] Fix miscelaneous bugs in the fleet setup page (#104479)

thomasneirynck committed:
[Maps] fix import (#104518)

flash1293 committed:
[Lens] Fix state setting via mutation (#104420)

Commit:
[Fleet][Logs UI] Ensure the <LogStream /> component reacts to query changes (#103876)

* Ensure the <LogStream /> component reacts to query changes

* Remove unused import

flash1293 committed:
improve debounce behavior (#104421)

jbudz committed:
[docs] Update default memory limit (#104535)

Co-authored-by: gchaps <33642766+gchaps@users.noreply.github.com>

renovate[bot] committed:
Update dependency @elastic/charts to v31.1.0 (#104455)

Co-authored-by: Renovate Bot <bot@renovateapp.com>

Commit:
[Fleet] Fix fn call (#104608)

scottybollinger committed:
[Workplace Search] Fix a couple of Source Display settings bugs (#104355)

* Remove `exact` prop on route

This was preventing the rendering of the subroute. Not sure why it was working before

* Replace `EuiFlexGrid` with `EuiFlexGroup`

Again, not sure why this was chosen, but although the EuiFlexGrid component renders fine in the standalone UI, it does not look correct in Kibana. EuiFlexGroup seems to render as intended.

scottybollinger committed:
[Workplace Search] Fix bug where error was behind modal stuck in loading state (#104360)

* Fix an issue from previous PR

In https://github.com/elastic/kibana/pull/104024, the error handling incorrectly used the `message` property on the response, when it should have been the attributes.errors array.

* Use inline error for duplicate name

darnautov committed:
[ML] fix marker positioning

elasticmachine commented:
Pinging @elastic/apm-ui (Team:apm)

elasticmachine commented:
Pinging @elastic/fleet (Team:Fleet)

darnautov commented:
Sorry for the noise!

kibanamachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/136892/)
* Commit: 694f6edcf52fae72c40f478fb8c0fff0bb911b28
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-104394/694f6edcf52fae72c40f478fb8c0fff0bb911b28)
* [Pipeline Steps](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/136892/flowGraphTable) (look for red circles / failed steps)
* [Interpreting CI Failures](https://www.elastic.co/guide/en/kibana/current/interpreting-ci-failures.html)

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
> ‼️ ERROR: build [0ceb6e54-86ef-4a41-aabb-ef5770baa36b] is not a valid complete build

### History
* :green_heart: [Build #136602](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/136602/) succeeded 1441b94adae980f9fa1d88cd3be1b6a31d65fff2
* :broken_heart: [Build #136486](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/136486/) failed 516cde6a5c3e4b41ea3b90b0b588d741564339cd

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

cc @darnautov

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/136486/","number":"136486","commit":"516cde6a5c3e4b41ea3b90b0b588d741564339cd"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/136602/","number":"136602","commit":"1441b94adae980f9fa1d88cd3be1b6a31d65fff2"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/136892/","number":"136892","commit":"694f6edcf52fae72c40f478fb8c0fff0bb911b28"}]}
PIPELINE-->
